Platform/AlderlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c

```diff
@@ -1128,10 +1128,10 @@ UpdateFspConfig (
       case PLATFORM_ID_ADL_N_DDR5_CRB:
         FspsConfig->PchLanEnable = 0x0;
         FspsConfig->PchPmVrAlert = 0x1;
-        FspsConfig->PchFivrExtV1p05RailEnabledStates = 0x3f;
+        FspsConfig->PchFivrExtV1p05RailEnabledStates = 0x1f;
         FspsConfig->PchFivrExtVnnRailEnabledStates = 0x3f;
         FspsConfig->PchFivrExtVnnRailSupportedVoltageStates = 0x4;
-        FspsConfig->PchFivrVccinAuxOffToHighCurModeVolTranTime = 0x0;
+        FspsConfig->PchFivrVccinAuxOffToHighCurModeVolTranTime = 0x96;
```

Comment:
@kvparikh89 seems like BIOS is still setting this to 150 by default. I remember they are supposed to revert to zero for PV release. Do you recall?

Reply:
Got some inputs from BIOS. This issue is resolved. 0x96 is good as a default.

Reply:
Thanks for clarifying